miden/src/lib.rs

```diff
@@ -19,3 +19,9 @@ pub use prover::{
     ProvingOptions, StackOutputs, Word,
 };
 pub use verifier::{verify, VerificationError};
+
+// (private) exports
+// ================================================================================================
+
+#[cfg(feature = "internal")]
+pub mod internal;
```

Comment:
Do we need this so that we can make benchmarks work?

Reply:
Yes basically the benchmarks (and integration tests in future PRs) want to access the `masm-examples` directory, and understand the file format. The `lib` target is the only place we can share code across all of them (i.e. there's no `dev-lib` which would be accessible only by the tests and benchmarks).

To emulate a "dev lib" target, I added this `internal` feature flag which users are not expected to use. Alternatively, we could simply expose `InputFile` too.

Reply:
I'm not a big fan of publishing the code which is used only for testing/benchmarking - so, I'd probably keep the `internal` feature.